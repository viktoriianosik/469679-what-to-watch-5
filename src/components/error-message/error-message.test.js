import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";


it(`Render component ErrorMessage correctly`, () => {
  const tree = renderer.create(
      <ErrorMessage
        message={`Error 404`}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
