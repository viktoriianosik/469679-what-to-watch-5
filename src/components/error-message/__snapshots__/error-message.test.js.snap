// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component ErrorMessage correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "red",
      "color": "white",
      "display": "flex",
      "height": "70px",
      "justifyContent": "center",
      "position": "absolute",
      "width": "100%",
      "zIndex": "100",
    }
  }
>
  Error 404
</div>
`;
